lib/features/family/features/reflect/presentation/widgets/recommendations_list_widget.dart

```diff
@@ -128,6 +137,39 @@ class _RecommendationsListWidgetState extends State<RecommendationsListWidget> {
             ),
           ),
         ),
+        const SizedBox(height: 24),
+        if (!widget.uiModel.isNotLoggedInParent ||
+            widget.uiModel.showActsOfService)
+          Padding(
+            padding: const EdgeInsets.symmetric(horizontal: 24),
+            child: FunButton(
+              onTap: () => widget.onRecommendationChosen?.call(_currentIndex),
+              text: widget.uiModel.showActsOfService
+                  ? "I'm going to do this"
+                  : 'Give',
+              analyticsEvent: AnalyticsEvent(
+                  AmplitudeEvents.newActOfGenerosityClicked,
+                  parameters: {
+                    widget.uiModel.showActsOfService
+                            ? 'act_of_service'
+                            : 'donation':
+                        widget.uiModel.organisations[_currentIndex].name,
+                  }),
+            ),
+          ),
+        const SizedBox(height: 16),
+        TextButton(
+          onPressed: () {
+            widget.onRecommendationChosen?.call(-1);
```

Comment:
Ah I see now where the -1 comes from. I would not do it this way. I recommend making a new fuction param in RecommendationsListWidget onSkip, and then cubit.onSkip. Then in cubit.onSkip you can still make use of onDeed.

Reply:
good point!